libs/scripts/delayed.js

```diff
@@ -1,7 +1,20 @@
-// eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './scripts.js';
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
 
-// Core Web Vitals RUM collection
-sampleRUM('cwv');
-
-// add more delayed functionality here
+/**
+ * Executes everything that happens a lot later, without impacting the user experience.
+ */
+export default async function delayed() {
```

Comment:
You either want to export or have line 20, not both. Line 20 is what is called a "side effect" and you don't want side effects and exports in the same file.

I would remove `export default` and you should be good. There's another way to do this, but I think what you have is cleaner.

Reply:
removed and fixed.